From 9fe89c9111a08add6d5d98262438f5aca334a0f0 Mon Sep 17 00:00:00 2001
From: finger <Larry.Finger@lwfinger.net>
Date: Tue, 14 May 2013 13:06:24 -0500
Subject: [PATCH 155/390] Change detection of missing proc routines away from
 kernel version

This fix makes a more robust checking of available features that can be
fooled by a stale include/linux/version.h

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 os_dep/os_intfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8192du/os_dep/os_intfs.c
+++ b/drivers/staging/rtl8192du/os_dep/os_intfs.c
@@ -247,7 +247,7 @@ static int	rtw_proc_cnt = 0;
 
 #define RTW_PROC_NAME DRV_NAME
 
-#if (LINUX_VERSION_CODE > KERNEL_VERSION(3, 8, 0))
+#ifndef create_proc_entry
 /* dummy routines */
 void rtw_proc_remove_one(struct net_device *dev)
 {
